daemon/cmd/status.go

```diff
@@ -521,6 +521,36 @@ func (c *clusterNodesClient) NodeConfigurationChanged(config datapath.LocalNodeC
 	return nil
 }
 
+func (c *clusterNodesClient) NodeNeighDiscoveryEnabled() bool {
+	// no-op
+	return false
+}
+
+func (c *clusterNodesClient) NodeNeighborRefresh(ctx context.Context, node nodeTypes.Node) {
+	// no-op
+	return
+}
+
+func (c *clusterNodesClient) NodeCleanNeighbors(migrateOnly bool) {
+	// no-op
+	return
+}
+
+func (c *clusterNodesClient) GetNodeIP(_ uint16) string {
+	// no-op
+	return ""
+}
+
+func (c *clusterNodesClient) DumpNodeIDs() []*models.NodeID {
+	// no-op
+	return nil
+}
+
+func (c *clusterNodesClient) RestoreNodeIDs() {
+	// no-op
+	return
+}
```

Comment:
Commit `datapath, ipcache: Remove unused node ID functions`, I believe the 6 functions above were backported by mistake.

Reply:
good catch :+1:

Reply:
there are a couple of commits having conflicts, however, this is the first time I am using backport tool, so didn't note down :(

Reply:
Managed to find the conflict commit in my terminal, I have updated the PR description with more details.

Reply:
Thanks Tam!

Yep, I don't use the backporter tool for exactly that reason. I like the text file of the old backporting scripts for note taking.